api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -1,9 +1,53 @@
 import typing
+from dataclasses import asdict
 
 import boto3
 from boto3.dynamodb.conditions import Key
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import Prefetch
+from flag_engine.django_transform.document_builders import (
+    build_identity_document,
+)
+
+from environments.models import Environment
+from features.models import FeatureState
+from features.multivariate.models import MultivariateFeatureStateValue
+
+from .types import DynamoProjectMetadata
+
+
+class DynamoProjectMetadataWrapper:
```

Comment:
I feel like the logic in this wrapper can absorbed into the `DynamoProjectMetadata` dataclass itself? We can use a classmethod to get a given metadata record from dynamo and load it into the dataclass.